src/canonmn_int.cpp

```diff
@@ -741,369 +763,457 @@ namespace Exiv2 {
 
     //! LensType, tag 0x0016
     extern const TagDetails canonCsLensType[] = {
```

Comment:
The diff of the following array looks like everything got changed, but when glancing over it I have a hard time to actually spot the changes. What happened here?

Reply:
The original code was pretty-formatted to align with lenses that have 4 spaces. Unfortunately, there came along lens #10001 and thus not aligned. I just gave ourselves more breathing room by using 5 spaces throughout.